src/App.tsx

```diff
@@ -1,9 +1,18 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const visibleMovies = moviesFromServer.filter((movie) => {
+    return (
+      movie.title.toLowerCase().includes(query.toLowerCase())
```

Comment:
```suggestion
      movie.title.toLowerCase().includes(query.toLowerCase().trim())
```
<img width="875" alt="Знімок екрана 2022-11-24 о 12 30 41" src="https://user-images.githubusercontent.com/106310796/203784076-5c624f05-f3ed-4660-861b-0a43507c2301.png">